substrate/client/chain-spec/src/lib.rs

```diff
@@ -338,6 +338,7 @@ pub use self::{
 		GenesisBlockBuilder,
 	},
 	genesis_config_builder::GenesisConfigBuilderRuntimeCaller,
+	json_patch::merge as json_merge,
```

Comment:
Maybe not most elegant solution, but I don't want to add crate for this single tiny `merge` function. Let me know if it is ok.